atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/impl/ExpectationsMetricsReportingTest.java

```diff
@@ -52,55 +66,63 @@ public void setUp() {
     @Test
     public void ageReported() {
         long age = 129L;
-        SnapshotTransaction.reportExpectationsCollectedData(age, BLANK_READ_INFO, BLANK_COMMIT_LOCK_INFO, metrics);
+        reportMetrics(BLANK_EXPECTATIONS_DATA.withAgeMillis(age));
         assertHistogramHasSingleValue(metrics.ageMillis(), age);
     }
 
     @Test
     public void bytesReadReported() {
         long bytes = 1290L;
-        reportMetricsWithReadInfo(BLANK_READ_INFO.withBytesRead(bytes));
+        reportMetrics(BLANK_EXPECTATIONS_DATA.withReadInfo(BLANK_READ_INFO.withBytesRead(bytes)));
         assertHistogramHasSingleValue(metrics.bytesRead(), bytes);
     }
 
     @Test
     public void kvsCallsReported() {
         long calls = 12L;
-        reportMetricsWithReadInfo(BLANK_READ_INFO.withKvsCalls(calls));
+        reportMetrics(BLANK_EXPECTATIONS_DATA.withReadInfo(BLANK_READ_INFO.withKvsCalls(calls)));
         assertHistogramHasSingleValue(metrics.kvsReads(), calls);
     }
 
     @Test
     public void mostKvsBytesReadInSingleCallNotReportedOnReadInfoWithEmptyMaximumBytesKvsCallOnFinishedTransaction() {
-        reportMetricsWithReadInfo(BLANK_READ_INFO);
+        reportMetrics(BLANK_EXPECTATIONS_DATA);
         assertThat(metrics.mostKvsBytesReadInSingleCall().getSnapshot().getValues())
                 .isEmpty();
     }
 
     @Test
     public void mostKvsBytesReadInSingleCallReportedOnReadInfoWithPresentMaximumBytesKvsCallOnFinishedTransaction() {
         long bytes = 193L;
-        reportMetricsWithReadInfo(
-                BLANK_READ_INFO.withMaximumBytesKvsCallInfo(ImmutableKvsCallReadInfo.of("dummy", bytes)));
+        reportMetrics(BLANK_EXPECTATIONS_DATA.withReadInfo(
+                BLANK_READ_INFO.withMaximumBytesKvsCallInfo(ImmutableKvsCallReadInfo.of("dummy", bytes))));
         assertHistogramHasSingleValue(metrics.mostKvsBytesReadInSingleCall(), bytes);
     }
 
     @Test
     public void commitLocksRequestsReported() {
-        SnapshotTransaction.reportExpectationsCollectedData(
-                0L,
-                BLANK_READ_INFO,
-                ImmutableTransactionCommitLockInfo.builder()
-                        .cellCommitLocksRequested(1)
-                        .rowCommitLocksRequested(2)
-                        .build(),
-                metrics);
+        reportMetrics(BLANK_EXPECTATIONS_DATA.withCommitLockInfo(ImmutableTransactionCommitLockInfo.builder()
+                .cellCommitLocksRequested(1)
+                .rowCommitLocksRequested(2)
+                .build()));
         assertHistogramHasSingleValue(metrics.cellCommitLocksRequested(), 1);
         assertHistogramHasSingleValue(metrics.rowCommitLocksRequested(), 2);
     }
 
-    private void reportMetricsWithReadInfo(TransactionReadInfo readInfo) {
-        SnapshotTransaction.reportExpectationsCollectedData(0L, readInfo, BLANK_COMMIT_LOCK_INFO, metrics);
+    @Test
+    public void metadataInfoReported() {
```

Comment:
Don't forget to update this when / if you change the name!

Reply:
I renamed any `metadataInfo...` to `writeMetadataInfo...`